src/widgets/WalletModal/ConnectModal.tsx

```diff
@@ -59,6 +57,34 @@ const getPreferredConfig = (walletConfig: Config[]) => {
   ];
 };
 
+const ScrollWrapper = styled(Box)`
+  overflow-x: hidden;
+  margin-right: 6px;
+  align-self: stretch;
+  ${({ theme }) => theme.mediaQueries.sm} {
+    margin-right: 14px;
+  }
+  &::-webkit-scrollbar {
+    width: 4px;
+  }
+  &::-webkit-scrollbar-track {
+    background-color: ${({ theme }) => theme.colors.gray200};
+    box-shadow: none;
+  }
+  &::-webkit-scrollbar-thumb {
+    background-color: ${({ theme }) => theme.colors.textSubtle};
```

Comment:
this is a old color, use new

Reply:
I don't know that type of color. Be more specific